src/pages/Homepage/LendByCategoryHomepage.vue

```diff
@@ -157,6 +157,47 @@
 				<h2 class="lender-quotes__header text-center small-12 columns">
 					What our lending <br class="so mo"> community thinks
 				</h2>
+				<div
+					class="lender-quotes__lender-quote--card small-10 medium-8 large-5 xlarge-4 columns"
+					v-for="lenderQuote in lenderQuotes"
+					:key="lenderQuote.name"
+				>
+					<kv-responsive-image
+						class="lender-quotes__lender-quote--lender-img"
+						:images="[
+							['small', lenderQuote.image],
+						]"
+						loading="lazy"
+						alt=""
+					/>
+					<div
```

Comment:
I'm not sure that these should be divs. To keep it semantic, I think they should be a combination of p's and spans. Is there any reason to use v-html here instead of just template interpolation {{ }} ?

Reply:
Nope no specific reason Eddie, thanks for bringing this up, I've made that change after reading a little on this. (https://forum.vuejs.org/t/performance-v-html-vs-interpolation-aka-braces/29559/3)

Reply:
@BoulderBrains 👍  theres also some XSS security issues with v-html, they wouldnt apply in this case, but something to be aware of: https://vuejs.org/v2/guide/syntax.html#Raw-HTML